core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -7429,6 +7456,13 @@ components:
       required: true
       schema:
         type: string
+    googleProjectIdPathParam:
+      name: googleProjectId
+      in: path
+      description: Google Project ID
+      required: true
+      schema:
+        type: string
```

Comment:
totally optional: you could add
```
        minLength: 6
        maxLength: 30
        pattern: '^[a-z0-9-]+$'
```
for nice client-side validation … in the swagger-ui only; this wouldn't change anything in the Scala tier